src/scripts/restore.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/env bash
+
+restore() {
+	if [ -z "${SKPR_BACKUP}" ]; then
+		SKPR_BACKUP=$(skpr backup list "${SKPR_SOURCE}" --json | jq -r '[.][0][0].Name')
+		if $? -gt 0; then
+			echo "Failed to get name of most recent backup"
+			exit 1
+		fi
+		# shellcheck disable=SC2086
+		echo "export SKPR_BACKUP=SKPR_BACKUP" >>$BASH_ENV
```

Comment:
Can you explain what this is doing? Looks like it's setting `SKPR_BACKUP` to `SKPR_BACKUP` (the string)

Reply:
Fixed up!

https://github.com/skpr/skpr-orb/blob/53bba37e8237d3d0a9a485985493a6602c780bac/src/scripts/restore.sh#L9-L13